packages/plugins/ansiblerole-insights-client/ansiblerole-insights-client.spec

```diff
@@ -0,0 +1,47 @@
+
+%global repo_orgname RedHatInsights
+%global repo_name insights-client-role
+%global repo_version 1.5
+
+%global role_orgname %{repo_orgname}
+%global role_name insights-client
+%global role_version %{repo_version}
+
+Name: ansiblerole-insights-client
```

Comment:
This seems sensible for naming. The only other example I could find was system roles packaging where they used `rhel-system-roles`.

https://access.redhat.com/articles/3050101